### DISCUSSION TITLE
Processing & Loading Data between two Deferent  DB Context

### DISCUSSION THREAD

**Discussion by AhmedJoum** [paillave/Etl.Net#405] [Q&A] [answered]
Is there any example of loading Data between two Deferent  DB Contexts?

**Comment by paillave** [comment, marked as answer]
Hello, I will have to make a clear documentation about it, but at the moment, the suject is covered in the issue https://github.com/paillave/Etl.Net/issues/399 (that I leave opened as long as the updated documentation is not published)
